tests/Tests/500__jupyterhub/test-minimal-image.robot

```diff
@@ -35,6 +35,21 @@ Can Spawn Notebook
     Fix Spawner Status
     Spawn Notebook With Arguments    image=s2i-minimal-notebook
 
+Verify Tensorflow Can Be Installed In The Minimal Python Image Via Pip
+    [Documentation]    Verify Tensorflow Can Be Installed In The Minimal Python image via pip
+    [Tags]    ODS-555
+    Open With JupyterLab Menu    File    New    Notebook
+    Sleep    1
+    Maybe Close Popup
+    Close Other JupyterLab Tabs
+    Maybe Close Popup
+    Sleep    1
```

Comment:
maybe you can try using `Launch A New Notebook Classic Notebook` from Jupyter library. @lugi0 thoughts?

Reply:
should it be the case to create a keyword to replace `Launch A New Notebook Classic Notebook`? I think these lines are used also in other places. If you think it's worth doing it @lugi0 @pranavsuryaw

Reply:
We could create a keyword, but what would you put in that keyword? IMHO it should not be any bigger than
```
Open New Notebook
    Open With JupyterLab Menu    File    New    Notebook
```
Maybe including also the sleep and `Maybe Close Popup` after. I'm not sure it's worth it to write a keyword to wrap around one or two lines.

Reply:
yeah i was thinking of wrapping lines from 41 to 43, and then from 44 to 46 (so two keywords), IF and only IF the two combination of lines are used somewhere else in the repo. 

If you agree, but you think it's too big change to apply in this PR, we could take notes and create a dedicated PR in the next sprint.

Reply:
I have created two keywords